server/eclipse-project/src/main/java/us/freeandfair/corla/query/CastVoteRecordQueries.java

```diff
@@ -401,9 +279,8 @@ public static CastVoteRecord get(final Instant the_timestamp,
       Main.LOGGER.error(COULD_NOT_QUERY_DATABASE);
     }
     if (result == null) {
-      Main.LOGGER.debug("found no CVR for timestamp " + the_timestamp + 
-                        ", county " + the_county_id + ", type " + 
-                        the_type);
+      Main.LOGGER.debug("found no CVR for county " + the_county_id +
```

Comment:
Should we include the sequence number in this message, since it was part of the query?

Reply:
Probably.

Reply:
It is included now, though oddly didn't show up near this comment.